src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -19,7 +21,12 @@ def to_representation(self, value):
         return value
 
     def to_internal_value(self, data):
-        if not isinstance(data, dict):
+        if isinstance(data, str):
+            try:
+                data = literal_eval(data)
+            except Exception:
+                raise ValidationError("Failed trying to parse dict from string")
```

Comment:
I'm not sure how safe this is: https://docs.python.org/3/library/ast.html#ast.literal_eval

It can't execute malicious code directly, but it seems like it might be possible to crash the server using this. How does the DRF serializer usually load this?

Not sure what the format is, but should we just be using `json.loads` or something?

Reply:
`json.loads` works after replacing the single quotes with double quotes